brave/brave-core
# Storybook: Added Error Handling to Wallet Locked Screen

Resolves <https://github.com/brave/brave-browser/issues/16027>

## Submitter Checklist:

1) Added Error Handling for the Password Input on the Wallet Locked Screen
2) Added Error Handling for the Password Input on the Wallet Locked Panel


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that storybook shows the new component.
3. Check functionality of the new component.

https://user-images.githubusercontent.com/40611140/121052893-0a6f4780-c778-11eb-9778-0fe628399ffd.mov
